apps/charterafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -0,0 +1,115 @@
+import Link from "@/commons-ui/next/Link";
+import { Typography, Box, Grid } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import React from "react";
+
+import OpportunityCard from "../OpportunityCard";
+
+import { neutral } from "@/charterafrica/colors";
+
+const OpportunityCardList = React.forwardRef(function OpportunityCardList(
+  props,
+  ref
+) {
+  const { title, grants } = props;
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const isSmallDesktop = useMediaQuery(theme.breakpoints.only("md"));
+  return (
+    <Box ref={ref}>
+      <Grid
+        container
+        justifyContent="space-between"
+        sx={{
+          marginBottom: "40px",
+        }}
```

Comment:
```suggestion
        rowSpacing={5}
```

... `Grid` [docs](https://mui.com/material-ui/react-grid/#row-amp-column-spacing)